src/workers/crypto.worker.ts

```diff
@@ -77,7 +79,7 @@ class CryptoWorker {
             this.platformUtilsService, this.logService);
 
         this.containerService = new ContainerService(this.cryptoService);
-        this.containerService.attachToGlobal(global);
+        this.containerService.attachToGlobal(self);
```

Comment:
again, im a little confused by this change

Reply:
See previous comment